### DISCUSSION TITLE
Create banner file in docker image?

### DISCUSSION THREAD

**Discussion by d-j-a** [drakkan/sftpgo#1279] [Q&A]
I'm attempting to create a banner file (banner.txt), but I'm having some issues..not sure if I am going about this the right way. 

I am running SFTPGo in a docker w/ UNRAID. I have found the file sftpgo.json in /etc/sftpgo

I attempted to edit the file, but vi or vim doesn't work. (doesn't exist??) I attempted to copy an edited version of the file into the directory to replace it- I don't have permission. I tried to su or sudo, I don't have the password and sudo doesn't even register as an option. 

Should I be adding a variable in the docker config outside of it to set this item? This seems like such a simple thing, but I can't sort it out, help! <please!>

**Comment by drakkan** [comment]
You can override settings via env vars, no need to change the default config file. I really appreciate your $2 personal donation, but please don't assume this means support for every question you ask :smile: 
Thanks for understanding
